### PR TITLE
Fix minibuffer completion style bug.

### PR DESCRIPTION
**Pull request by 7ym0n**
see #354

### CODE REVIEW

File: lisp/init-dict.el [hunk 1]
```diff
@@ -98,7 +98,7 @@
                            :max-width (/ (frame-width) 2)
                            :max-height (/ (frame-height) 2)
                            :background-color (face-background 'tooltip nil t)
-                           :internal-border-color (face-background 'posframe-border nil t)
```
Comment (seagle0128) [inline lisp/init-dict.el:101]: Why change the faces?

File: lisp/init-lsp.el [hunk 2]
```diff
@@ -222,7 +222,7 @@
        :init (setq lsp-ui-sideline-show-diagnostics nil
                    lsp-ui-sideline-ignore-duplicate t
                    lsp-ui-doc-delay 0.1
-                   lsp-ui-doc-border (face-background 'posframe-border nil t)
```
Comment (seagle0128) [inline lisp/init-lsp.el:225]: Same as above.

**Review by seagle0128 — CHANGES_REQUESTED** [review]

**Comment by seagle0128** [comment]
It was fixed in cab662ce8d0a0dfb10d383f55824a3f3d83fa448.

**Comment by 7ym0n** [comment]
set completion style for `minibuffer'.

childframe-workable-p is false, it's not work.

**Pull request closed without merging**